### PR TITLE
Added new ImageGen async class. Added quiet flag to avoid cli cluttering. Added spanish README

### PR DESCRIPTION
**Pull request by scmanjarrez**

### CODE REVIEW

**Review by acheong08 — CHANGES_REQUESTED** [review]
Needs to be different class. We want both a sync and async version. The async version can extend the sync

**Comment by scmanjarrez** [comment]
Okay, I have reverted ImagenGen changes and added async to a new class called ImageGenAsync, is that good?

File: src/ImageGen.py [hunk 1]
```diff
@@ -263,12 +273,18 @@ def main(args: argparse.Namespace) -> None:
...
+    if not args.asyncio:
+        # Create image generator
+        image_generator = ImageGen(args.U, args.quiet)
+        image_generator.save_images(
+            image_generator.get_images(args.prompt),
+            output_dir=args.output_dir,
+        )
     else:
-        main(args)
+        asyncio.run(async_image_gen(args))
```
Comment (acheong08) [inline src/ImageGen.py:290]: The default should be sync

**Review by acheong08 — CHANGES_REQUESTED** [review]

**Comment by acheong08** [comment]
Sorry for the slow response. My email got very cluttered

**Comment by scmanjarrez** [comment]
I'm currently working with the update, sorry.

**Review by acheong08 — APPROVED** [review]

**Pull request merged**